solaris/packer/package_generation/vagrant/Vagrantfile

```diff
@@ -108,8 +108,10 @@ Vagrant.configure("2") do |config|
     os.each do |key, value|
         config.vm.define "#{key}" do |node|
           node.vm.box = "#{value['box']}"
-          if key == "solaris10"
-            node.vm.box_url = 'https://packages-dev.wazuh.com/utils/vagrant/solaris/10/solaris10.box'
+          if key == "solaris11"
+            node.vm.box_url = 'https://s3-us-west-1.amazonaws.com/packages-dev.wazuh.com/utils/vagrant/solaris/11/i386/solaris11.3.box'
+          elsif key == "solaris10"
+            node.vm.box_url = 'https://s3-us-west-1.amazonaws.com/packages-dev.wazuh.com/utils/vagrant/solaris/10/solaris10.box'
```

Comment:
Same here

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/191/commits/49424d4fb30ffd2950a5e8cb0d1fac8c69fdf893